packages/xod-arduino/test-cpp/dtoa.cpp

```diff
@@ -0,0 +1,64 @@
+
+// Uncomment for debug:
+// #include <iostream>
+#include "catch.hpp"
+#include "../platform/types.h"
+#include "../platform/listViews.h"
+#include "../platform/listFuncs.h"
+#include "../platform/dtoa.h"
+#include "XStringFormat.inl"
```

Comment:
You’re looking for `XStringFormat.inl` elsewhere, should be:

```
#include <XStringFormat.inl>
```

And better put it before the local includes.